manifests/state-whereabouts-cni/0050-whereabouts-ds.yaml

```diff
@@ -54,7 +54,7 @@ spec:
           effect: NoSchedule
       containers:
       - name: whereabouts
-        image: {{ .CrSpec.Repository }}/{{ .CrSpec.Image }}:{{ .CrSpec.Version }}
+        image: {{ .CrSpec.Repository }}/{{ .CrSpec.Image }}:{{ .CrSpec.Version }}{{ .RuntimeSpec.ImageSuffix }}
```

Comment:
Do we need this suffix thing? It looks like [whereabouts image](https://github.com/k8snetworkplumbingwg/whereabouts/pkgs/container/whereabouts/99256749?tag=v0.6.2) is already multiarch

Reply:
good catch, I'll remove this change